incubator/command-config/src/test/resources/io/aklivity/zilla/runtime/command/config/internal/openapi/http/proxy/plain/zilla.yaml

```diff
@@ -0,0 +1,51 @@
+name: example
+bindings:
+  tcp_server0:
+    type: tcp
+    kind: server
+    options:
+      host: 0.0.0.0
+      port: 8080
+    exit: http_server0
+  http_server0:
+    type: http
+    kind: server
+    options:
+      access-control:
+        policy: cross-origin
+    routes:
+    - exit: http_client0
+      when:
+      - headers:
+          :path: /items
+          :method: GET
+    - exit: http_client0
+      when:
+      - headers:
+          :path: /items
+          :method: POST
+    - exit: http_client0
+      when:
+      - headers:
+          :path: /items/*
+          :method: GET
+    - exit: http_client0
+      when:
+      - headers:
+          :path: /items/*
+          :method: PUT
+    - exit: http_client0
+      when:
+      - headers:
+          :path: /items/*
+          :method: DELETE
+  http_client0:
+    type: http
+    kind: client
+    exit: tcp_client0
+  tcp_client0:
+    type: tcp
+    kind: client
+    options:
+      host: "${{env.TCP_CLIENT_HOST}}"
+      port: "${{env.TCP_CLIENT_PORT}}"
```

Comment:
I don't think we need to quotes for this one since `port` is normally integer type.

Reply:
We build the json object, then apply the json patch, then generate the yaml from the json result. So, this output is generated by `YAMLMapper` and it quotes these env var strings because they have all kinds of funky characters. We have a few options to tweak, but I don't think we can fine tune the yaml generation to this extent. However, if this actually turns out to be a problem for zilla, then we'll need to find a way to solve it after all. 🤔

https://github.com/attilakreiner/zilla/blob/2814d370eb0f9de900bfdd500af810048ef4643b/runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/config/ConfigWriter.java#L116-L119

Reply:
So it turns out that the quoting doesn't cause any problems and zilla correctly parses this to an int array anyway. As this would be pretty cumbersome to fix (e.g. applying some custom regexp to the generated output), my preference is to leave it as is, pls LMK.